pages/in/embed/_id/list.vue

```diff
@@ -0,0 +1,204 @@
+      <like-form>
+        <template slot="header-right">
+          <a
+            class="lc-underline"
+            href="https://like.co/"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            About LikeCoin
+          </a>
+        </template>
+
+        <span class="lc-color-like-dark-brown-2 lc-font-size-20">
+          {{ numOfLikes }} Like from {{ likees.length }} people
+          <span v-if="title">{{ title }}</span>
+        </span>
+
+        <div
+          :class="['likee-list', 'lc-margin-top-8', { expand: isShowAll }]"
+          :style="{ maxHeight: `${Math.ceil(likees.length / 2) * 74}px` }"
+        >
+
+          <div
+            v-for="(likee, index) in likees"
+            :key="index"
+            class="user-avatar"
+          >
+            <img
+              v-if="likee.avatar"
+              :src="likee.avatar"
+            >
+            <nuxt-link
+              :to="{ name: 'id', params: { id: likee.id } }"
+            >
+              <span v-if="likee.displayName">{{ likee.displayName }}</span>
+              <span v-else>{{ likee.id }}</span>
+            </nuxt-link>
+          </div>
+
+          <transition name="lc-transition-default">
+            <div
+              v-if="!isShowAll"
+              class="overflow-overlay"
+            />
+          </transition>
+        </div>
+
+        <div
+          class="lc-margin-top-12 lc-text-align-center"
+        >
+          <md-button
+            v-if="!isShowAll"
+            class="lc-color-like-green"
+            @click="isShowAll = true"
+          >
+            show more
+          </md-button>
+        </div>
+      </like-form>
+    </div>
+  </div>
+</template>
+
+
+<script>
+import Vue from 'vue'; // eslint-disable-line import/no-extraneous-dependencies
+import axios from '~/plugins/axios';
+import LikeForm from '~/components/LikeForm';
+import {
+  apiGetUserMinById,
+  apiGetLikeButtonLikerList,
+  apiGetLikeButtonTotalCount,
+} from '@/util/api/api';
+
+export default {
+  name: 'embed-id-list',
+  layout: 'narrowWithHeader',
+  components: {
+    LikeForm,
+  },
+  async asyncData({ params, query }) {
+    const promises = [
+      apiGetLikeButtonLikerList(params.id, query.referrer),
+      apiGetLikeButtonTotalCount(params.id, query.referrer),
+    ];
+    if (query.referrer) {
+      let url = query.referrer;
+      const mediumRegEx = '^(?:https?:\\/\\/)?medium\\.com\\/media\\/[a-zA-Z0-9_]+\\?postId=([a-zA-Z0-9_]+)';
+      const match = query.referrer.match(mediumRegEx);
+      if (match && match[1]) url = `https://medium.com/p/${match[1]}`;
+      promises.push(axios.get(url).catch(() => ''));
+    }
+    const [{ data: likees }, { data: totalData }, { data: html }] = await Promise.all(promises);
+    let title = '';
+    if (html) {
+      let match = html.match('<title>(.*?)</title>');
+      if (match && match[1]) [, title] = match;
+      match = html.match('"title":"(.*?)"');
```

Comment:
Won't work with titles like `"title":"\"quoted words\""`.
Also, escaped characters will be weird, since the `\` is not removed.
Need to really parse the JSON response instead of 'hacking' the content out from the response string.

Reply:
but seems is this best effort extraction, I dont really know the structure of json
and parsing an arbitrary source of json sounds dangerous to me